packages/dds/sequence/src/test/fuzz/fuzzUtils.ts

```diff
@@ -406,7 +407,7 @@ export const defaultFuzzOptions: Partial<DDSFuzzSuiteOptions> = {
 		clientAddProbability: 0.1,
 	},
 	defaultTestCount: 100,
-	saveFailures: { directory: path.join(__dirname, "../../../src/test/fuzz/results") },
+	saveFailures: { directory: path.join(_dirname, "../../../src/test/fuzz/results") },
```

Comment:
To use the _dirname from parent, the relative path here will need to be one ../ less.